Doc/library/sqlite3.rst

```diff
@@ -792,10 +804,8 @@ Connection objects
 
       Register callable *authorizer_callback* to be invoked for each attempt to
       access a column of a table in the database. The callback should return
-      :const:`SQLITE_OK` if access is allowed, :const:`SQLITE_DENY` if the entire SQL
-      statement should be aborted with an error and :const:`SQLITE_IGNORE` if the
-      column should be treated as a NULL value. These constants are available in the
-      :mod:`!sqlite3` module.
+      one of :const:`SQLITE_OK`, :const:`SQLITE_DENY`, or :const:`SQLITE_IGNORE`
+      to signal how the column should be handled.
```

Comment:
The latter seems better, since unitl now it wasn't entirely obvious to me that these constant are returned by the user to tell SQLite what to do -- I thought it was the opposite.

Reply:
Yeah, that seems a fair bit better to me; I rewrote this several times and wasn't really happy with it in this very regard. Added the latter.